gymnasium_robotics/envs/multiagent_mujoco/coupled_half_cheetah.py

```diff
@@ -0,0 +1,116 @@
+import os
+
+import gymnasium
+import numpy as np
+from gymnasium.envs.mujoco import mujoco_env
+from gymnasium.utils.ezpickle import EzPickle
+
+DEFAULT_CAMERA_CONFIG = {
+    "distance": 4.0,
+}
+
+
+class CoupledHalfCheetah(mujoco_env.MujocoEnv, EzPickle):
+    """This environment consists of 2 half cheetahs coupled by an elastic tendon."""
+
+    metadata = {
+        "render_modes": [
+            "human",
+            "rgb_array",
+            "depth_array",
+        ],
+        "render_fps": 20,
+    }
+
+    def __init__(self, render_mode: str = None):
+        """Init.
+
+        Args:
+            render_mode: see [Gymansium/MuJoCo](https://gymnasium.farama.org/environments/mujoco/)
+        """
+        self._forward_reward_weight = 1
```

Comment:
How are these values decided

Reply:
They are the same defaults as their simpler equivalent